Asterix/phase_amplitude_functions.py

```diff
@@ -2,12 +2,13 @@
 import skimage.transform
 from astropy.io import fits
 import Asterix.processing_functions as proc
+import Asterix.fits_functions as useful
 
 
 ##############################################
 ##############################################
 ### Pupil
-def roundpupil(dim_pp, prad):
+def roundpupil(dim_pp, prad, no_pixel=False):
```

Comment:
what does no_pixel do? Grey pupil ? If so, change the name to grey_pupil?
Complete the description.

Reply:
ca fait un rescaling d'une pupille de 6K pixel à la taille qui faut. J'ai pas eu le temps de faire les tests, pour le moment c'est inactif car par defaut a false et jamais utilisé. J'ai ajouté une description. 

J'avoue que j'ai été un peu surpris de ta méfiance de cette manipulation, ça me parait rapide et relativement innofensif et ça réduit pas mal les erreurs numériques